app/src/main/java/org/stepik/android/view/step/ui/fragment/StepFragment.kt

```diff
@@ -131,6 +131,8 @@ class StepFragment : Fragment(R.layout.fragment_step), StepView,
                 )
         }
 
+        stepContentNext.isVisible = stepWrapper.step.position < lessonData.lesson.steps.size && !stepWrapper.isStepCanHaveQuiz
```

Comment:
In which cases can be `stepWrapper.step.position >= lessonData.lesson.steps.size`?

Reply:
`stepWrapper.step.position` indexing starts from 1, so it is possible that `stepWrapper.step.position == lessonData.lesson.steps.size`